packages/blocks/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 6.3.0 (Unreleased)
```

Comment:
Should we stub out 6.2.0 ? I think I recall @youknowriad mentioning something about whether it's worth the effort to add in. Conversely, I think someone working in the code might only look at the versions they see in the CHANGELOG and bump according to that, without considering what's actually published to npm.

Reply:
yes, we should update the package changelog management docs accordingly.

Reply:
I had a hard time picking the correct version. I admit that having a simple `## 6.2.0 (xx-xx-xx)` would make it less error-prone. Other contributors might not take an effort and look in `package.json` file.

Reply:
I think the suggestion was to replace `## 6.3.0 (Unreleased)` with 
`## x.x.x (Unreleased)` or something like that.

Reply:
🙃 Updating again

Reply:
I guess then I may have told @jorgefilipecosta the wrong thing at https://github.com/WordPress/gutenberg/pull/14179#discussion_r269778501 😅